aws/table_aws_redshift_subnet_group.go

```diff
@@ -0,0 +1,182 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/redshift"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsRedshiftSubnetGroup(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_redshift_subnet_group",
+		Description: "AWS Redshift Subnet Group",
+		Get: &plugin.GetConfig{
+			KeyColumns:        plugin.SingleColumn("cluster_subnet_group_name"),
+			ShouldIgnoreError: isNotFoundError([]string{"ClusterSubnetGroupNotFoundFault"}),
+			Hydrate:           getRedshiftSubnetGroup,
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listRedshiftSubnetGroup,
+		},
+		GetMatrixItem: BuildRegionList,
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "cluster_subnet_group_name",
+				Description: "The name of the cluster subnet group.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "description",
+				Description: "The description of the cluster subnet group.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "subnet_group_status",
+				Description: "The status of the cluster subnet group.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "vpc_id",
+				Description: "The VPC ID of the cluster subnet group.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "subnets",
+				Description: "A list of the VPC Subnet elements.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "tags_src",
+				Description: "A list of tags attached to the subnet group.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getRedshiftSubnetGroup,
+				Transform:   transform.FromField("Tags"),
+			},
+
+			// Standard columns
+			{
+				Name:        "tags",
+				Description: resourceInterfaceDescription("tags"),
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.From(getRedshiftSubnetGroupTurbotTags),
+			},
+			{
+				Name:        "title",
+				Description: resourceInterfaceDescription("title"),
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ClusterSubnetGroupName"),
+			},
```

Comment:
Move `tags` column after `title`.
Generally we add the standard column in this order(i.e. `title`, `tags` and `akas`)